servers/quarkus-server/src/test/java/org/projectnessie/server/BaseClientAuthTest.java

```diff
@@ -43,8 +44,8 @@ protected NessieApiV1 api() {
       return api;
     }
 
-    HttpClientBuilder builder =
-        HttpClientBuilder.builder().withUri("http://localhost:19121/api/v1");
+    String quarkusUrl = Objects.requireNonNull(System.getProperty("test.url"));
+    HttpClientBuilder builder = HttpClientBuilder.builder().withUri(quarkusUrl + "/api/v1");
```

Comment:
nit: I know we can't extend this class with `AbstractTestQuarkusRest`, but is it possible for all these classes (that have these duplicate codes) to call API building from a unified place instead of duplicating the same code?

Reply:
added a utils class